syft/frameworks/torch/tensors/interpreters/replicated_shared.py

```diff
@@ -48,7 +48,7 @@ def __arrange_workers(workers):
 
     def generate_shares(self, plain_text, number_of_shares=3):
```

Comment:
If ```generate_shares``` is public shouldn't we move the ```__validation``` function here?
I should also remove the line from 51?

Reply:
it's not part of the interface, I left it public just to test it